docs/examples/plot_linear_quickstart.py

```diff
@@ -52,9 +52,25 @@
 ######################################################################
 # ``preds`` holds the decision values, i.e. the raw values
 # outputted by the model. To transform it into predictions,
-# the simplest way is to take the positive values as
-# the labels predicted to be associated with the sample,
-# i.e. ``preds > 0``.
+# the simplest way is to take the positive values as the labels predicted
+# to be associated with the sample, i.e. ``preds > 0``. 
+
+preds[preds > 0] = 1
+preds[preds <= 0] = 0
+
+######################################################################
+# Now we have labels predicted in binary form. Next step,
+# use the label mapping in the ``Preprocessor`` to transform it to multiclass form.
```

Comment:
We now have the label mask. Next, we use `label_mapping` in `Preprocessor` to get the original labels.